WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -1309,13 +1309,11 @@ extension NotificationDetailsViewController {
         }
 
         notificationCommentDetailCoordinator?.onSelectedNoteChange = self.onSelectedNoteChange
-        weak var navigationController = navigationController
 
-        dismiss(animated: true, completion: {
```

Comment:
As far as I can tell, the `dismiss` does nothing on iPhone, but it does break notifications if they are presented modally.